Users/Install.rst

```diff
@@ -83,10 +83,80 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is the most user friendly method of managing virtual environment and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for our project, move to your project
+directory and run the following command.
+
+::
+
+    $ pipenv --three
+
+The above command will create virtual environment isolated from your project
```

Comment:
I think `isolated from your project directory` is incorrect. The concept is same as virtualenv, like `isolated from system site packages`, or `isolated from global Python installation`

Reply:
The point of using `pipenv` is to provide a more user-friendly tool to the user for both their `pip` and `virtualenv` use cases and hence reduces one dependency to take care of. Also, it's now [recommended](https://packaging.python.org/new-tutorials/installing-and-using-packages/#installing-pipenv) by python.org as well.

Reply:
@mamat-rahmat is right here, isolated from environment/system packages/global installation is the right context here